netutils/ip.py

```diff
@@ -596,3 +597,33 @@ def get_usable_range(ip_network: str) -> str:
         lower_bound = str(net[1])
         upper_bound = str(net[-2])
     return f"{lower_bound} - {upper_bound}"
+
+
+def sort_list_ips(ips: t.Union[str, t.List[str]], sort_type: str = "network") -> str:
+    """Given a concatenated list of CIDRs sorts them into the correct order and returns as concatenated string.
+
+    Args:
+        ips (t.Union[str, t.List[str]]): Concatenated string list of CIDRs, IPAddresses, or Interfaces or list of the same strings.
+        sort_type (str): Whether the passed list are networks, IP addresses, or interfaces, ie "address", "interface", or "network".
+
+    Returns:
+        str: Sorted concatenated list of sort_type IPs.
+    """
+    if sort_type not in ["address", "interface", "network"]:
+        raise ValueError("Invalid sort type passed. Must be `address`, `interface`, or `network`.")
+    if isinstance(ips, list):
+        ips_list = ips
+    elif (isinstance(ips, str) and "," not in ips) or not isinstance(ips, str):
+        raise ValueError("Not a concatenated list of IPs as expected.")
+    elif isinstance(ips, str):
+        ips_list = ips.replace(" ", "").split(",")
+
+    functions = {"address": ipaddress.ip_address, "interface": ipaddress.ip_interface, "network": ipaddress.ip_network}
```

Comment:
```suggestion
    functions: t.Dict[str, Callable] = {"address": ipaddress.ip_address, "interface": ipaddress.ip_interface, "network": ipaddress.ip_network}
```

Maybe try this to resolve the error

Reply:
Thank you, that's what I was thinking might be needed.